lib/git-hooks.js

```diff
@@ -3,6 +3,12 @@ var util = require('util');
 var spawn = require('child_process').spawn;
 var fs = require('fs');
 var fsHelpers = require('./fs-helpers');
+var execSync = require('child_process').execSync;
+if (!execSync) {
+    execSync = require('runsync').exec;
```

Comment:
I do love sync version but I want to support `0.8` and `0.10` as well without extra modules. Let's make async version.
```javascript
excludeIgnoredPaths(hooks, function (filteredHooks) {
    runHooks(filteredHooks, [arg], callback);
})
```